manifests/volume.pp

```diff
@@ -74,7 +74,7 @@
 
   $_transport = "transport ${transport}"
 
-  if $options and ! empty( $options ) {
+  if ! empty( $options ) {
```

Comment:
Later there's another if $_options. Perhaps you can combine those?

Reply:
I think we should leave this pull request as it is, because many others depend on it and we need a working CI pipeline.